### ISSUE TITLE
Preferred tooling for profile definition

### ISSUE THREAD

**Issue by Dunmail** [HL7-UK/UK-Core-Access#3]
What tooling do we use for profile definition?

Propose:
https://build.fhir.org/ig/HL7/fhir-shorthand/
https://github.com/FHIR/sushi

**Comment by riksmithies** [comment]
yes by far the best choice. We should consider rewriting UK code in FSH.

**Comment by riksmithies** [comment]
I mean "core"

**Comment by clarotech** [comment]
I am already creating FSH versions of the UK Core content, more as a use-case for other discussions I am having with options for developing profiles. In my opinion, it is definitely the way to go when creating profiles.

**Comment by Dunmail** [comment]
Sushi project committed for review at first meeting

**Comment by Dunmail** [comment]
Two options for publishing IG when finished.
1. HL7 Tooling
2. Simplifier

At this stage, no need to commit as we can generate FHIR conformance resources. These can be loaded to Simplifier as FHIR resources for consistency with UK Core. Need to consider whether implementers who wish to derive product-specific capability statement are able to do this without using Simplifier.

Documentation (e.g. for search parameters) can be provided as markdown files within this repository.

**Comment by Dunmail** [comment]
FSH and Sushi agreed as preferred tooling for profile definition.

Separate issue created for IG publication: https://github.com/HL7-UK/UK-Core-Access/issues/6
